### ISSUE TITLE
Custom height for TSMessageView

### ISSUE THREAD

**Issue by kartikthapar** [KrauseFx/TSMessages#211]
Is there a way to customize the message view height? I want the height of the message view to be same as that of a navigation bar, i.e. 44px.

**Comment by EmmanuelC19** [comment]
same here!

**Comment by anwhat** [comment]
+1

**Comment by ridvank** [comment]
Is there any solution?

**Comment by trungnguyen1791** [comment]
+ 1.
I want smaller height like Facebook Message app.

**Comment by tonchis** [comment]
+1

**Comment by tbaranes** [comment]
+1

**Comment by argoel** [comment]
+1

**Comment by gothic1234** [comment]
+1

**Comment by tbaranes** [comment]
+1

**Comment by aehlke** [comment]
Ideally we can just supply our own custom view, with its own height.

**Comment by anneTaterTot** [comment]
+ 1

**Comment by anneTaterTot** [comment]
+1

**Comment by johnnypayso** [comment]
+1

**Comment by staufman** [comment]
One option is to set the `TSMessageViewProtocol` delegate on the `sharedMessage` instance `TSMessage`. Then, implement `- (void)customizeMessageView:(TSMessageView *)messageView;`. Because this library still uses springs/struts, you should be able to customize the view to your heart's content without having to worry about layout constraints that you don't manage.

**Comment by SandeepAggarwal** [comment]
@staufman There is no public property available of TSMessageView to adjust its height.

**Comment by staufman** [comment]
@SandeepAggarwal It has been a while now but since `TSMessageView` is just a `UIView`, you should just be able to change its `frame` right?

**Comment by SandeepAggarwal** [comment]
@staufman Yes , you are right but still the frame remains unchanged and the  `TSMessageView` just shifts downwards upon changing its height.

**Comment by staufman** [comment]
The fact that something _is_ changing means it's most likely something in your view hierarchy that is causing the problem. Without knowing more about your particular codebase, I'm not sure what, offhand, would be causing this.

**Comment by aafa** [comment]
```
[TSMessage setDelegate:self];

- (void)customizeMessageView:(TSMessageView *)messageView {
	[messageView mas_makeConstraints:^(MASConstraintMaker *make) {
		make.height.equalTo(@44);
	}];
}
```
or whatever height you want

tested on ios 8 and 9
